Mapify/SceneInitializers/GameContent/WorldMapSetup.cs

```diff
@@ -17,10 +17,13 @@ public class WorldMapSetup : SceneSetup
         private static bool modifiedMaterial;
         private static Texture defaultTexture;
         public static bool ShowStationNamesOnMap;
+        private static Material mapMaterial;
 
         public override void Run()
         {
-            MapLifeCycle.OnCleanup += () => modifiedMaterial = false;
+            //restore base map texture on exiting a custom map
+            MapLifeCycle.OnCleanup += () => modifiedMaterial = false; mapMaterial.mainTexture = defaultTexture;
```

Comment:
Does this event get cleared anywhere? Not a bug introduced in this PR, bug if not then this will get called an additional time every time Run is called

Reply:
No it did not get cleared. I got rid of the Action so now it definitely only gets called once.